site/gatsby-site/plugins/graphql-codegen-babel/index.js

```diff
@@ -0,0 +1,5 @@
+// Gatsby doesn't support importing babel plugins without default export so we need to export it as default
```

Comment:
nit: prefer .ts for new files related to type checking, especially auxiliary/util files that will get imported.

Reply:
I'm not sure if that would work in this particular case because this happens before Babel is set up to transpile, but I agree otherwise.